nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -461,4 +482,18 @@ public DocumentRef getCommentedDocumentRef(CoreSession s, DocumentModel commentD
         });
 
     }
+
+    @Override
+    protected boolean hasComments(CoreSession session, DocumentModel document) {
+        String query = String.format( //
+                QUERY_GET_COMMENTS_COUNT_FOR_DOCUMENT_BY_COMMENT_ANCESTORS, document.getId());
+        return !session.queryProjection(query, 1, 0).isEmpty();
+    }
+
+    @Override
+    protected boolean hasComments(CoreSession session, DocumentModel document, String author) {
```

Comment:
```suggestion
    protected boolean hasComments(CoreSession session, DocumentModel document, String user) {
```
To be consistent with the interface.